packages/mobile_app/lib/components/magic_poll.dart

```diff
@@ -3,9 +3,15 @@ import 'package:flutter_svg/flutter_svg.dart';
 
 import 'package:mutable_wrappers/mutable_wrappers.dart';
 
+/// it is faster to call empty function
+/// then check if function == null
+/// for onTap method
+void _dummyFunc() {}
+
 class MagicPollPicker extends StatefulWidget {
+  final Function onTap;
   final MutableInteger value;
-  MagicPollPicker({@required this.value});
+  MagicPollPicker({@required this.value, this.onTap = _dummyFunc});
```

Comment:
is it possible to have smth like (pseudocode) `{@required this.value, this.onTap = () => {}}`?

Reply:
Dart dont allow it in constructor =(